tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -1059,3 +1064,146 @@ def register(
         image_transform = inverse_translation @ tissue_transform @ forward_translation
 
         return image_transform @ transform_initializer
+
+
+def estimate_bspline_transform(
+    fixed_image: np.ndarray,
+    moving_image: np.ndarray,
+    fixed_mask: np.ndarray,
+    moving_mask: np.ndarray,
+    grid_space: float = 50.0,
+    scale_factors: list = (1, 2, 5),
+    shrink_factor: list = (4, 2, 1),
+    smooth_sigmas: list = (4, 2, 1),
+    num_iterations: int = 100,
+    sampling_percent: float = 0.2,
+):
+    """Estimate BSpline transformation.
+
+    This function performs registration using multi-resolution Bspline approach.
+
```

Comment:
Is there some reference we can put in here to read more about this method? If not can we explain a bit more about what is happening?

Reply:
I have added a link to the SimpleITK documentation:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/fd991ca60f990452bcc8ee22b7c149e65544d376/tiatoolbox/tools/registration/wsi_registration.py#L1073-L1081